include/zephyr/drivers/espi_emul.h

```diff
@@ -33,45 +32,49 @@ extern "C" {
 
 struct espi_emul;
 
+/* Forward-Declare from emul.h */
+struct emul;
```

Comment:
See above comment on preventing recursive inclusion. So we forward declare just this one thing.